x-pack/plugins/infra/server/features.ts

```diff
@@ -14,11 +14,11 @@ export const METRICS_FEATURE = {
   order: 700,
   icon: 'metricsApp',
   navLinkId: 'metrics',
-  app: ['infra', 'kibana'],
+  app: ['infra', 'metrics', 'kibana'],
   catalogue: ['infraops'],
```

Comment:
Maybe a silly question: why is this PR updating the infra app and not the apm app?

Reply:
APM is already "compliant" with the new requirement that your `navLinkId` value must be included in your `app` array. Infra was one of the few that weren't already configured this way:

https://github.com/elastic/kibana/blob/2642d658c4a94fc77017ad425a442cd65baa2521/x-pack/plugins/apm/server/feature.ts#L16-L17